ts/features/idpay/initiative/timeline/components/TimelineRefundDetailsComponent.tsx

```diff
@@ -93,11 +93,16 @@ const TimelineRefundDetailsComponent = (props: Props) => {
           {I18n.t("idpay.initiative.operationDetails.refund.resultLabel")}
         </Body>
         <IOBadge
-          small={true}
-          labelColor={
+          small
+          variant={
+            refund.operationType === OperationTypeEnum.REJECTED_REFUND
+              ? "outline"
+              : "solid"
+          }
+          color={
             refund.operationType === OperationTypeEnum.REJECTED_REFUND
               ? "red"
-              : "bluegreyDark"
+              : "aqua"
```

Comment:
@mastro993 a little bit refactor to this to use the new `IOBadge` variant. This could be verbose a little bit. what do you think?

Reply:
That's right. No problem for the verbosity, we have only two cases so we can't do any better.